spec/utils-spec.coffee

```diff
@@ -83,3 +83,15 @@ describe 'Brainstem Utils', ->
       it 'sets a function on options.error', ->
         expect(options.error).toBeDefined()
         expect(options.error).toEqual(jasmine.any(Function))
+
+  describe '.chunk', ->
+    it 'returns an array of arrays of specified length', ->
+      expect(Utils.chunk([1, 2, 3, 4], 2)).toEqual [[1, 2], [3, 4]]
+
+    context 'when array is not an array', ->
+      it 'returns an empty array', ->
+        expect(Utils.chunk(null, 2)).toEqual []
+
+    context 'when count is null or 0', ->
+      it 'returns empty', ->
+        expect(Utils.chunk([1, 2], null)).toEqual []
```

Comment:
One thing we haven't tested is what happens when count is greater than the array's length.